app/ItemsJSON.txt

```diff
@@ -0,0 +1,42 @@
+[
+	{
```

Comment:
Added a basic Items JSON so retrofit has something to consume

Reply:
.txt works, though I'd end it with the .json extension.